rfc-001-meme.md

```
@@ -0,0 +1,254 @@
+  "exps":  [12, 0, 0, 4, 1, 2, 0, 0, 0, 0, 1, 0, 0, 1, 1],
+  "frame": 140,
+  "index": 14,
+  "order": 4
+}
+```
+Implementations **SHOULD** include this vector in their conformance test suite
+and check that re-encoding reproduces the same bytes (and hence the same CID).
+## 4. Content addressing
+### 4.1 CID of a meme block
+The CID of a meme block is the CIDv1
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)